crates/frontend/src/pages/function/function_create.rs

```diff
@@ -26,7 +26,7 @@ pub fn create_function_view() -> impl IntoView {
                     window.editor = monaco.editor.create(document.querySelector('.monaco'), {{
 
                         value: [
-                            'async function validate() {{',
+                            'async function validate(value, key) {{',
```

Comment:
Let us add a default return value to the template function as well
```suggestion
                            'async function validate(value, key) {{',
                            
                            return true;
                            }}'
```

Reply:
The syntax of the suggestion might be off @mahatoankitkumar  - but you possibly got the point.  Basically want the template to convey that this function has to return a bool.